utils/src/main/kotlin/OrtProxySelector.kt

```diff
@@ -171,6 +177,12 @@ class OrtProxySelector(private val fallback: ProxySelector? = null) : ProxySelec
     override fun select(uri: URI?): List<Proxy> {
         requireNotNull(uri)
 
+        if (proxyIncludes.isNotEmpty()) {
+            if (proxyIncludes.none { it.isNotEmpty() && (uri.authority.endsWith(it) || uri.host.endsWith(it)) }) {
```

Comment:
For readability and de-deplication of `it.isNotEmpty() && (uri.authority.endsWith(it) || uri.host.endsWith(it)` I believe it's beneficial to extract it to a function.

Reply:
Done.